lib/commands/general.js

```diff
@@ -110,7 +110,7 @@ commands.getWindowRect = async function () {
 };
 
 commands.hideKeyboard = async function (strategy, ...possibleKeys) {
-  if ((this.opts.deviceName || '').indexOf('iPhone') === -1) {
+  if (!(this.opts.deviceName || '').includes('iPhone')) {
```

Comment:
is that already handled in the other PR?

Reply:
Yes. This was done before I did that.